### DISCUSSION TITLE
Changelog grouped by week/day rather than version?

### DISCUSSION THREAD

**Discussion by alecthomas** [orhun/git-cliff#737] [Q&A]
We release on every commit, which makes the default of grouping changes by tag quite noisy. Is there a way to group by day or week? I dug into the context a bit, but nothing really seemed obvious. I tried `group_by(attribute="timestamp")` without success.

Something like the following:

## Week of 2024-07-01 (v0.254.3 - v0.256.7)

Features

- ...
- ...

Bug Fixes

- ...
- ...

etc.

**Comment by orhun** [comment]
We would need to create a custom Tera filter for this and it is not super straightforward. Maybe leave a comment [here](https://github.com/Keats/tera/issues/637)
